.github/workflows/release-please.yml

```
@@ -0,0 +1,21 @@
+  pull-requests: write
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Release Please
+        uses: googleapis/release-please-action@v4
+        with:
+          release-type: node
+          package-name: ${{ github.event.repository.name }}
+          # For monorepos, switch to manifest mode with a config file.
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)